src/braket/jobs/_entry_point_template.py

```diff
@@ -1,8 +1,16 @@
 run_entry_point = """
 import cloudpickle
-from braket.jobs import save_job_result
+import os
+from braket.jobs import get_results_dir, save_job_result
 from braket.jobs_data import PersistedJobDataFormat
 
+
+# set working directory to results dir
+os.chdir(get_results_dir())
+
+# create symlinks to input data
+link_input()
```

Comment:
Can we prevent the input data from getting populated in the results? Since the "recovered()" function encapsulates the entirety of the job script, could we unlink the input data in our template after that function finishes execution? This would ensure that the function has had a chance to consume the input data, but remove the input data before the results are packaged.

Reply:
Sure, we can add that. I can make sure that each link is a symlink and points to the original destination before deleting, in case it has been overwritten explicitly by the user.